homeassistant/components/solarlog/manifest.json

```
@@ -0,0 +1,8 @@
+{
+  "domain": "solarlog",
+  "name": "Solar-Log",
+  "documentation": "https://www.home-assistant.io/components/solarlog",
```

We just changed all the urls ! :) 

```suggestion
  "documentation": "https://www.home-assistant.io/integration/solarlog",
```

---

Changed